### PR TITLE
Allow users to specify tab order.

### LINKED ISSUE #14: Touchbar Editing
Is there a possibility of adding a bar to change what shows up on the Touch Bar? For example, I only want spec, inventory, prayer and mage to show up so I don't have to search around and can keep my fingers in the same spot.

### PR DESCRIPTION
**Pull request by vartan**
The config page looks fairly different after this change, since each "tab" gets its own ConfigSection with two entries. 
The new config box allows users to specify a tab order number from 0-13; the default settings match the order before this change. 

This is my first RuneLite related pull request and I'm unfamiliar with conventions, so I am very open to suggestions if anything seems off to you.

This would resolve #14 

Config screenshot with reversed order:
image
Corresponding Touchbar screenshots:
Touch Bar Shot 2023-07-16 at 8 48 52 PM
Touch Bar Shot 2023-07-16 at 8 49 45 PM

### CODE REVIEW

File: build.gradle
Comment (vartan) [inline build.gradle:1]: I couldn't get the plugin to show up when using `runeLiteVersion = '1.8.25'`. If this is a problem for some reason, i can revert it.

**Comment by PatrickKocken** [comment]
Hi @vartan,

Thanks for the pull request! I will take a look in RuneLite to see if there is something like a reorderable list for ordering the tab. Additionally, I wasn't aware that `latest.release` could be used for the `runeLiteVersion` in the build.gradle.

The code looks good to me. Could you please change the destination branch for the pull request from `master` to `development`? I believe the RuneLite plugin hub bot monitors the master branch for changes and automatically generates a pull request to update the plugin whenever it detects any changes.

Thanks!

**Comment by vartan** [comment]
> I will take a look in RuneLite to see if there is something like a reorderable list for ordering the tab

right? I tried looking through the `net.runelite.client.config` package but didn't see anything. If you find something I can update the PR. 

> Could you please change the destination branch for the pull request from master to development

Is this good now? I see it now includes a lot of other changes that were in `master` but not `development`. Sorry, I rarely use git (I work on a large monorepo backed by perforce, basically)

**Comment by vartan** [comment]
image

it appears this is not supported at the moment

EDIT: also runelite does not monitor master branch

**Comment by PatrickKocken** [comment]
I merged the PR to master. Tonight I'll check the plugin and make a PR to update the plugin in RuneLite's plugin hub

**Pull request merged**